test/test_binary_ufuncs.py

```diff
@@ -88,6 +89,64 @@ def _make_tensor(shape, dtype, device, fill_ones=False) -> torch.Tensor:
 
 # TODO: update to use opinfos consistently
 class TestBinaryUfuncs(TestCase):
+    @ops(binary_ufuncs)
+    def test_broadcasting(self, device, dtype, op):
```

Comment:
These broadcasting tests can probably just run in float32 by doing:

```
@ops(binary_ufuncs, allowed_dtypes=(torch.float32,))
```

Reply:
There's a small potential flaw in this approach, which @heitorschueroff realized earlier, if an operator doesn't support float32 because the test will be skipped. But let's cross that bridge if/when we get there